src/move.h

```diff
@@ -532,6 +532,7 @@ class ParallelTempering : public Move {
     std::unique_ptr<MPI::Partner> partner;     //!< Policy for finding MPI partners
     Geometry::VolumeMethod volume_scaling_method = Geometry::VolumeMethod::ISOTROPIC; //!< How to scale volumes
     std::map<MPI::Partner::PartnerPair, Average<double>> acceptance_map;              //!< Exchange statistics
+    std::map<int, std::vector<MPI::Partner::PartnerPair>> exchange_map;              //!< Exchange statistics
```

Comment:
The memory requirements of the data structure scales with number of steps and could be an issue for long simulations. Perhaps better to simply let each replica stream to a file?

Reply:
That is true. I thought about streaming to a file at first, but did not get it to work.. I will look into it again.